src/QueryEditor.tsx

```diff
@@ -35,18 +38,36 @@ export const QueryEditor: React.FC<Props> = ({ query, datasource, onChange, onRu
     handleQueryChange({ ...query, columns }, false);
   };
 
+  const loadTableOptions = _.debounce((query: string, cb?: LoadOptionsCallback<string>) => {
+    datasource
+      .queryTables(query)
+      .then((tables) => cb?.(tables.map((t) => ({ label: t.name, value: t.id, description: t.id }))))
+      .catch(handleError);
+  }, 500);
```

Comment:
Should we lower the debounce to 250ms?

https://stackoverflow.com/questions/42361485/how-long-should-you-debounce-text-input#:~:text=One%20important%20number%20to%20keep,keep%20your%20site%20feeling%20responsive.

Reply:
Based on that SO post and playing around with it some, I lowered the interval to 300ms.